.travis.yml

```diff
@@ -10,6 +10,8 @@ env:
   matrix:
     
     - CONDA_PY=27
+    - CONDA_PY=34
+    - CONDA_PY=35
```

Comment:
Appears this was an [error]( https://github.com/conda-forge/fake-factory-feedstock/pull/2/files#r94264111 ).